package.json

```diff
@@ -15,7 +15,10 @@
   },
   "author": "Benjamin Plouzennec <plouzennec.benjamin@gmail.com>",
   "license": "GNU GPL",
-  "dependencies": {},
+  "dependencies": {
+    "rollup-plugin-uglify": "^2.0.1",
```

Comment:
Shoudn't theses belong in the devDependencies ? Till the built library wont use them ?